Misc/NEWS.d/next/Windows/2019-07-29-16-49-31.bpo-37702.Lj2f5e.rst

```diff
@@ -0,0 +1 @@
+Fix memory leak in ssl certification.
```

Comment:
It might be good to specify which methods this applies to, and that it is only on Windows.

Reply:
I'd rather not descript c-implementation level information.